Dart/src/com/jetbrains/lang/dart/ide/runner/server/vmService/DartVmServiceListener.java

```diff
@@ -129,6 +136,8 @@ private void onIsolatePaused(@NotNull final IsolateRef isolateRef,
       final boolean suspend = myDebugProcess.getSession().breakpointReached(xBreakpoint, logExpression, suspendContext);
       if (!suspend) {
         myDebugProcess.getVmServiceWrapper().resumeIsolate(isolateRef.getId(), null);
+      } else {
+        myDebugProcess.getSession().positionReached(suspendContext);
```

Comment:
`breakpointReached()` is already called 3 lines above, why do you think you need to call `positionReached()`?